bin/domain_classification.R

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env Rscript
```

Comment:
Please add license information here - if you've created this script specifically for the pipeline (looks like it could be), MIT is fine. Else please  use the license where the code has been copied from.

Nice example/tempalte when taken from elsewhere: https://github.com/nf-core/viralintegration/blob/master/bin/chimeric_contig_evidence_analyzer.py

Example when written for the pipeline: https://github.com/nf-core/eager/blob/dev/bin/filter_bam_fragment_length.py#L3-L4